pango/src/auto/font.rs

```diff
@@ -27,6 +33,28 @@ glib::wrapper! {
 
 impl Font {
     pub const NONE: Option<&'static Font> = None;
+
+    #[cfg(any(feature = "v1_50", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+    #[doc(alias = "pango_font_deserialize")]
+    pub fn deserialize(
+        context: &Context,
+        bytes: &glib::Bytes,
+    ) -> Result<Option<Font>, glib::Error> {
```

Comment:
Can this return `None` without an error?

Reply:
As mentioned in prior comments, it can return None yes

Reply:
How very confusing, thanks :)